BaragonUI/app/models/Group.coffee

```diff
@@ -76,6 +76,7 @@ class Group extends Model
                 vex.dialog.buttons.NO
             ]
             callback: (data) =>
+                return if data is false
```

Comment:
rather than doing this, you can just add the `required` attribute to the input tag for the form